pybossa/util.py

```diff
@@ -1178,11 +1178,30 @@ def description_from_long_description(desc, long_desc):
     description = blank_space_regex.sub(" ", text_desc)
     return description if description else " "
 
-def get_user_type(firm_num):
-    firm_to_type = current_app.config.get('FIRM_TO_TYPE', {})
-    return firm_to_type.get(int(firm_num), None) if firm_num else None
+def generate_bsso_account_notification(user, admins_emails, access_type, warning=False):
+    template_type = 'adminbssowarning' if warning else 'adminbssonotification'
+    template = '/account/email/{}'.format(template_type)
 
-def get_user_data_access_level(firm_num):
-    firm_to_type = current_app.config.get('FIRM_TO_TYPE', {})
-    return ['L2'] if firm_num and int(firm_num) in firm_to_type.keys() else ['L4']
+    is_qa = current_app.config.get('IS_QA')
+    server_url = current_app.config.get('SERVER_URL')
+    brand = current_app.config.get('BRAND')
+
+    subject = 'A new account has been created via BSSO for {}'.format(brand)
+    msg = dict(subject=subject,
+               recipients=admins_emails)
+    try:
+        fullname = user['fullname']
+    except:
+        fullname = user['firstName'][0] + " " + user['lastName'][0]
 
+    msg['body'] = render_template('{}.md'.format(template),
```

Comment:
do we need to add `.md` or `.html` when rendering the template?

Reply:
For `msg['body']`, `.md` is used. For `msg['html']`, `.html` is used.